var/spack/repos/spack_repo/builtin/packages/py_dask_expr/package.py

```diff
@@ -19,7 +19,12 @@ class PyDaskExpr(PythonPackage):
 
     depends_on("python@3.10:", type=("build", "run"))
     depends_on("py-setuptools@62.6:", type="build")
-    depends_on("py-versioneer@0.28+toml", type="build")
+
+    # Strictly, this should be 0.28. However others in the dask ecosystem
+    # require 0.29, which makes it fail to concretize. Since Versioneer 0.29 doesn't
+    # break anything with 0.28, it should be safe to keep this aligned
+    # https://github.com/python-versioneer/python-versioneer/releases/tag/0.29
+    depends_on("py-versioneer@0.28: +toml", type="build")
```

Comment:
Should we guard against a potential future 0.30 breaking things?
```suggestion
    depends_on("py-versioneer@0.28:0.29 +toml", type="build")
```

Reply:
This comment applies to other instances as well.

Reply:
I don't think so. It is a slow moving project and it hasn't really had many breaking changes (if any really?)
https://github.com/python-versioneer/python-versioneer/blob/master/UPGRADING.md

I think leaving it floating and deal with issues if they come up is best.